org.eclipse.xtext.ide.tests/src/org/eclipse/xtext/ide/tests/server/ValidatorTest.xtend

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
+ * Copyright (c) 2019 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.ide.tests.server
+
+import org.eclipse.xtext.ide.tests.testlanguage.validation.TestLanguageValidator
+import org.junit.Test
+
+import static org.junit.Assert.*
+
+import static extension org.eclipse.xtext.util.Strings.*
+import org.eclipse.xtext.diagnostics.Diagnostic
+
+/**
+ * @author Christian Dietrich - Initial contribution and API
+ */
+class ValidatorTest extends AbstractTestLangLanguageServerTest {
+
+	@Test
+	def void testMultilineDiagnostic() {
+		'MyType1.testlang'.writeFile('''
+			type Multiline_Demo {
+			}
```

Comment:
Could you please add another test that has more than one char in the endline but different start- and end lines?